docs/changehistory/NextVersion.md

```diff
@@ -2,11 +2,34 @@
 
 Table of contents:
 
-- [@itwin/components-react](#itwincomponents-react)
+- [@itwin/appui-react](#itwinappui-react)
   - [Deprecations](#deprecations)
+  - [Additions](#additions)
+  - [Changes](#changes)
+- [@itwin/components-react](#itwincomponents-react)
+  - [Deprecations](#deprecations-1)
 - [@itwin/core-react](#itwincore-react)
   - [Fixes](#fixes)
 
+## @itwin/appui-react
+
+### Deprecations
+
+- Deprecated `StatusBar` component in favor of `StatusBarComposer`. [#848](https://github.com/iTwin/appui/pull/848)
+- Deprecated `StatusBar` namespace. Please use the components directly. [#848](https://github.com/iTwin/appui/pull/848)
+  - Deprecated `StatusBar.Popup` and `StatusBarPopup` in favor of `StatusBarPopover`.
+  - Deprecated `StatusBar.Field` and `StatusBarField` in favor of [iTwinUI Button](https://itwinui.bentley.com/docs/button).
+- Deprecated `StatusBarIndicator` in favor of [iTwinUI Button](https://itwinui.bentley.com/docs/button) (or other components) and AppUI `StatusBarPopup`. [#848](https://github.com/iTwin/appui/pull/848)
```

Comment:
`StatusBarPopup` is deprecated, just point to `StatusBarPopover` directly.

Reply:
My mistake, I wrote the wrong thing. Fixed now.